pycbc/types/timeseries.py

```diff
@@ -93,7 +93,7 @@ def sample_rate_close(self, other):
         if isinstance(other, TimeSeries):
             odelta_t = other.delta_t
         else:
-            odelta_t = other
+            odelta_t = 1.0/other
```

Comment:
I don't understand this line. What are we expecting `other` to be in this instance. I think maybe an inline comment and/or including the method's help text would help here.

Reply:
I've added a comment to indicate that this to handle either giving a time series to check if delta_t is close or giving an explicit sample rate (i.e. a number).

Reply:
To clarify, this didn't handle giving a float before at all.